api/src/controllers/station.controller.ts

```diff
@@ -0,0 +1,172 @@
+import {
```

Comment:
All of these endpoints are generated. We only focus on `GET /stations`. Feel free to add any comments